src/nuxeo-cloud/what-is-nuxeo-cloud.md

```diff
@@ -0,0 +1,208 @@
+<p data-renderer-start-pos="4398">No</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4404">No</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4410">No</p>
+</div>
+</td>
+</tr>
+<tr>
+<td rowspan="1" colspan="1">
+<p data-renderer-start-pos="4425">Logs via Security Dashboard</p>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4456">Yes</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4463">Yes</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4470">No</p>
+</div>
+</td>
+</tr>
+<tr>
+<td rowspan="1" colspan="1">
+<p data-renderer-start-pos="4597">PCI / SOC2 Compliant</p>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4621">Yes</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4628">Yes</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4635">No</p>
+</div>
+</td>
+</tr>
+<tr>
+<td rowspan="1" colspan="1">
+<p data-renderer-start-pos="4485">SSH / Infrastructure Access</p>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4516">No</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4522">No</p>
+</div>
+</td>
+<td rowspan="1" colspan="1">
+<div class="fabric-editor-block-mark sc-bwCtUz iXkdcO" data-align="center">
+<p data-renderer-start-pos="4528">No</p>
+</div>
+</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+## Monitoring and Support
+
+All environments are proactively monitored 24x7 and supported by the Nuxeo Cloud team.
+
+[More info&nbsp;<i class="fa fa-long-arrow-right" aria-hidden="true"></i>]({{page page='support'}})
+
+## Continuous Improvement
+
+The Nuxeo Cloud deployment technologies use Nuxeo best practices and are continuously improved upon in coordination with the Nuxeo Platform.
+
+[More info&nbsp;<i class="fa fa-long-arrow-right" aria-hidden="true"></i>]({{page page='deployment'}})
+
+## Security
+
+All environments are enabled with out of the box application and infrastructure level security monitors. The security model is continuously reviewed and today Nuxeo Cloud is both SOC2 and PCI/DSS Compliant.
```

Comment:
change to 

All Production and Pre-Production environments are enabled with application and infrastructure level security monitors. The security model is continuously reviewed by the Nuxeo Security team. As of 2019, the Nuxeo Cloud Production and Pre-Production environments are SOC2 and PCI/DSS Compliant.